### DISCUSSION TITLE
Creation of table without `File` column

### DISCUSSION THREAD

**Discussion by moashxer** [RafaelGB/obsidian-db-folder#647] [Ideas]
### Present your request

Hi, 

First of all thanks a lot for this amazing plugin. 

You know that obsidian tables aren't convenient at all on using tables. And this feature is amazing, however don't limit it to `Database Folder` kindly add the feature **to use this plugin also without need of file `Rows`.** Still this feature can be used to take notes or use without files but when creating each row it creates a new file hence leads on creating more files in the folder unnecessarily

Features in short

- Ability use as usual table without file column
- No creation of file when adding row 
- Or all the rows in same file

Kindly consider this feature.
Thanks for your amazing work :)

### For which platform do you request this request??

Cross

**Comment by RafaelGB** [comment]
Hello @moashxer ,
There was a plugin with that intention => https://github.com/trey-wallis/obsidian-notion-like-tables

Looks like its not maintaining anymore but could be helpful for you anyway. This plugin has its architecture oriented to metadata,  this feature in my opinion escapes from the solution we propose.

We can try to find another developer to maintain the repository I share at start, I think it has a lot of potential

**Reply by moashxer** [reply]
Yes, I'm aware of this plugin, after using it I decided to request here due to limited settings on that plugin and with some very useful features with your plugin in database settings itself.

**Comment by RafaelGB** [comment]
I will more your idea to discussion and I will be happy to help with another dev using this plugin interface

**Reply by moashxer** [reply]
Regarding this case I'm not aware of any dev to request to maintain. 

Apart from the useful features from your plugin, when comparing the both plugins, the only main difference is that this `db-folder` plugin makes **must to have files** and the `notion like tables` don't, if this limitation could be optional it would possibly fix the need.

However I respect your decision which you stick to.

Thanks :)

**Reply by RafaelGB** [reply]
Adapting dbfolder to traditional table format requires a lot of work currently, I don't want to rule the option out entirely in the distant future

Thanks for your ideas @moashxer and the use of this plugin

**Reply by moashxer** [reply]
Didn't know about the hard work that you had to put through it. When there is no files would it also requires much work to put all the rows to **one single file** instead of **each individual file?** 

Thanks to you for the good work and update.

**Reply by RafaelGB** [reply]
It would require a lot of work because it is not just moving it in a single row, but adapting it to markdwon's own tables, creating a new editing engine for it, a different reading engine, and also configuring it to switch from one engine to another.

I wanted to create the bases for it on csv files, once mounted we could look at it for this case but it will take time
